hello/GeneralDrivingPlaylist.py

```diff
@@ -66,31 +66,44 @@
         tracks_uri.append(x['uri'])
         # print(x['name'])
         # print(x['popularity'])
-    
-# function to narrorw down search to match our workout sound attributes
-def drivingTracks_uri():
-    for tracks in tracks_uri: #look through searched tracks uri
-        selected_tracks_data = spotifyObject.audio_features(tracks) #get audio features of searched track
-        for track_data in selected_tracks_data:
-            if 100 <= track_data['tempo'] <= 200:
-                selectedDrivingTracks_uri.append(track_data['uri'])
-            elif 0.5 <= track_data['valence'] <= 1.0:
-                selectedDrivingTracks_uri.append(track_data['uri'])
-            elif 0.4 <= track_data['liveness'] <= 1.0:
-                selectedDrivingTracks_uri.append(track_data['uri'])
-    return selectedDrivingTracks_uri
-    
-    
 
 
-def savePlaylist():
-
-    for x in range(0,30):
-        randsongs.append(selectedDrivingTracks_uri[random.randint(0,(len(selectedDrivingTracks_uri) - 1))])     #Fill randsongs list with  30 randomly selected songs from playlist list
+class drivingPlaylistClass():
```

Comment:
This looks great, this will help to organize code and improve the readability and functionality of our code.